items.md

```diff
@@ -586,3 +586,26 @@ Return codes:
 * 401 Unauthorized - if you are not authenticated
 * 403 Forbidden - if you are not logged in with sufficient permissions
 * 404 Not found - if the item doesn't exist (or was already deleted)
+
+## Search methods
+
+### findDuplicates
+
+**GET /api/core/items/search/findDuplicates?uuid=<:uuid>**
```

Comment:
This endpoint is not aligned to our REST best practices. A search method in the items collection resource endpoint must return a list of Item Resource.
If we need this endpoint it should be moved to a new endpoint specific for duplicate groups but as the feature is intended to be used only in the submission I don't expect this to be really needed

Reply:
I have moved this to a separate controller

Reply:
@kshepherd : This REST Contract doesn't appear to be updated with your new endpoint/controller.  Could you update it please?

Also, take a look at my feedback on the backend PR, as I suggested you might rename this to be something like..
```
/api/submission/duplicates/search?uuid=
```

That is modeled similar to the Controlled vocabulary endpoints which are also used primarily in Submission: https://github.com/DSpace/RestContract/blob/main/vocabularies.md